web/templates/caseload/index.html.eex

```diff
@@ -0,0 +1,17 @@
+<h2 class="tc">Caseload</h3>
+
+<%= for user <- @hl_users do %>
+  <div class="w-100 hl-bg-grey pa3 mv2">
+    <p class="b di"><%= user.name %></p><span class="fr">></span>
+  </div>
+<% end %>
+
+<%= for user <- @non_hl do %>
+  <div class="w-100 ph3 pv2">
+    <p>
+       <p class="b di"><%= user."Forename" %> <%= user."Surname" %></p>
```

Comment:
Was just reading: http://blog.plataformatec.com.br/2016/05/ectos-insert_all-and-schemaless-queries/

Might be another approach to this.